2.train-VAE/3.beta_vae_training_extract_model_components.ipynb

```diff
@@ -9,7 +9,7 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    extracted_col = metadata['ModelID']
</pre><p>nice, dynamic line of code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>